workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Apps/AppCreationFlow/ProvideVersionDetailsPage/ProvideVersionDetailsPage.tsx

```diff
@@ -93,12 +96,24 @@ export function ProvideVersionDetailsPage({
 			({key}) => key === 'dxp-license-usage-type'
 		);
 
-		const targetOption = isDXP ? dxpOption : trialOption;
+		const cloudOption = options.find(
+			({key}) => key === 'cloud-license-usage-type'
+		);
```

Comment:
What do you think to change this `options.find(({key}) => key ===""` into a function?
```
const findOptionByKey = (key: string) => options.find(({ key: k }) => k === key);

const trialOption = findOptionByKey('trial');
const dxpOption = findOptionByKey('dxp-license-usage-type');
const cloudOption = findOptionByKey('cloud-license-usage-type');
```